zk-token-sdk/src/sigma_proofs/grouped_ciphertext_validity_proof.rs

```diff
@@ -111,7 +111,7 @@ impl ValidityProof {
         }
     }
 
-    /// The ciphertext validity proof verifier.
+    /// The grouped ciphertext validity proof for 2 handles verifier.
```

Comment:
```suggestion
    /// Verifies a grouped ciphertext validity proof for 2 handles.
```
I was struggling to figure out how to hyphenate to disambiguate the modifiers for `verifier`. If you like this suggestion, it might be nice to rewrite all the `verify()` docs similarly with respect to the actions they take.

Reply:
Yes you are right. I will also update the constructor docs to be consistent with this.

Reply:
I updated the constructor docs for the `grouped_ciphertext_validity_proof`. I'll create a follow-up PR to update the others.